src/js/services/whatsapp.js

```diff
@@ -64,6 +64,6 @@ module.exports = function(shariff) {
       'tr': 'Whatsapp\'ta paylaş',
       'zh': '在Whatsapp上分享'
     },
-    shareUrl: 'whatsapp://send?text=' + encodeURIComponent(title) + '%20' + url + shariff.getReferrerTrack()
+    shareUrl: 'https://wa.me/?text=' + encodeURIComponent(title) + '%20' + url + shariff.getReferrerTrack()
```

Comment:
Why not using instand [`https://api.whatsapp.com/send?text=A Text`](https://api.whatsapp.com/send?text=A%20Text). 
`wa.me` is a redirect to `api.whatsapp.com`, just for shorter urls. So if we use `api.whatsapp.com` the redirection to the app is faster.

Reply:
[https://github.com/bradvin/social-share-urls#whatsapp](https://github.com/bradvin/social-share-urls#whatsapp) says the same. There are also a few further links to references and documentation. And there is a test note:
> Test Notes : wa.me does not work unless a phone_number is supplied, so, for this reason, we are suggesting users use the api.whatsapp.com format.
> Test Results : Functionality tested and verified by SocialShare Dev's in 2020.

@philwolstenholme If you are still available: Do you think we should do so? If so, could you change your PR? We should then ping the maintainers so they merge it. Otherwise, if you are not available anyore, let us know so someone else can make a PR.

Reply:
Hi everyone, I agree we should use `https://api.whatsapp.com` — I think when I made this PR I didn't realise the longer URL was available. I won't have time to work on this now, so if someone else could pick it up that would be great 👍

Reply:
@NoNamePro0 Do you want to make a pull request? If not, I can do that. Just let me know here.

Reply:
Yes. I can do it 👍